### ISSUE TITLE
Add descriptive alt text to avatar image

### ISSUE THREAD

**Issue by Aditya948351** [FireFistisDead/pdf-qa-bot#542]
The avatar image currently has an empty alt attribute. It should have a descriptive alt text for better accessibility.

**Comment by Aditya948351** [comment]
@Aditya948351 :- I am done With the PR review and merge and close this issue.
